CAKE-autorate.sh

```diff
@@ -24,13 +33,37 @@ cleanup_and_killall()
 	exit
 }
 
+# check to see if the named interface is present; exit if not
+is_if_present()
+{
+    the_if=$1
+    is_here=$(ifconfig | grep "$the_if")
+    if [ "$is_here" == "" ]; then
+        log_msg "'$the_if' interface not present... Aborting!\n"
+		log_msg "Install SQM QoS and configure CAKE-config.sh\n"
+        cleanup_and_killall
+    fi
+}
+
 install_dir="/root/CAKE-autorate/"
 
-. $install_dir"config.sh"
+. $install_dir"CAKE-config.sh"
```

Comment:
Shouldn't we name CAKE-config.sh rather CAKE-autorate-config.sh? I ask because the config is very much for CAKE-autorate and not so much for CAKE itself, if this makes sense?

Reply:
Probably, yes. And change the installer to CAKE-autorate-setup.sh